.env.example

```diff
@@ -39,3 +39,15 @@ BLOCKED_DOMAINS=evil.corp,bad.instance
 # If you enable this, instead of @user@twitter.com, the user will be @user@<ALTERNATIVE_TWITTER_DOMAIN>
 USE_ALTERNATIVE_TWITTER_DOMAIN=true
 ALTERNATIVE_TWITTER_DOMAIN=twitter.activitypub.actor
+
+# Database settings
+# Postgres
+DB_HOST=localhost
+DB_PORT=5432
+DB_NAME=mastodon_twitter_poster_production
+DB_USER=mastodon_twitter_poster
+DB_PASS=
+
+# Redis only for Sidekiq
+REDIS_PROVIDER=REDIS_URL
```

Comment:
What is this used for?

Reply:
`DB_PASS` is for setting the password of postgresql connection.
`REDIS_PROVIDER=REDIS_URL` is for indicate that REDIS_URL is the env var for setting redis connection for sidekiq, I' m not sure there is a default for this in sidekiq.

Reply:
Yeah, the default is REDIS_URL, I think you can remove this one :)

Reply:
Done !